src/index.css

```diff
@@ -55,6 +55,7 @@ a:hover {
   padding-left: 10%;
   padding-right: 10%;
   padding-bottom: 25px;
+  padding-top: 25px;
```

Comment:
If we add this style change here, won't it change the behavior of the a tag everywhere in the application?  I believe the a tag is also being used in several other places.  I think maybe have this as an in line change, or specify it in the file specific CSS path?

Reply:
I believe the way the change shows in GitHub makes it look like the property was changed in the "a" tag. But this was actually for the body div, as there was not enough space (especially on smaller screens) between the logo/menu buttons and the lesson title. Here are some images to show the change; However, let me know if I should revert:
![image](https://github.com/YomenT/bluebird-teaching-frontend/assets/94801110/f3c9ee63-c886-4075-b344-6375f8ea6eb0)
![image](https://github.com/YomenT/bluebird-teaching-frontend/assets/94801110/03d613ff-15e4-447e-8ae7-8e930f29dcaa)